src/components/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -323,14 +323,14 @@ export default class PeopleAndRoles extends Mixins(CommonMixin) {
           await this.confirmReassignCp().then(confirm => {
             if (confirm) {
               // remove the other person's CP role and their email
-              otherPerson.roles = otherPerson.roles.filter(r => r.roleType !== Roles.COMPLETING_PARTY)
-              otherPerson.officer.email = null
+              otherPerson.roles = otherPerson.roles.filter(r => r.roleType !== RoleTypes.COMPLETING_PARTY)
+              delete otherPerson.officer.email
```

Comment:
This deletes the property so that I can compare the objects (since one object may have email=null and the other object may not have email at all).

Reply:
I've never seen this syntax before!

Reply:
Very cool, useful for deleting object properties i see!

Reply:
This definitely beats a `null` definition. 
Noted!